src/mono/wasm/runtime/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "dotnet-runtime",
```

Comment:
Is it convention to have a package.json like this if this package is never meant to be uploaded to NPM?

Reply:
It is a project file essentially, it drives tooling and install dependencies.
On uploading it to npm, I'm not sure if this package it the one we will publish, when nodeJS port is ready. I guess 50% chance.